datanode/gateway/graphql/resolvers.go

```diff
@@ -910,7 +921,7 @@ func (r *myQueryResolver) EstimateOrder(
 
 	return &OrderEstimate{
 		Fee:            &fee,
-		TotalFeeAmount: ttf,
+		TotalFeeAmount: (mfee.Add(ifee).Add(lfee)).String(),
```

Comment:
Is a test possible ?

Reply:
it's a pain to be fair, this API is calling to others APi and aggregate stuff out of it. For just a addition that sounds overdone.